src/sleep/sleep.c

```diff
@@ -239,36 +237,42 @@ static int execute_s2h(usec_t hibernate_delay_sec) {
         if (r < 0)
                 return r;
 
-        r = rtc_read_time(&original_time);
-        if (r < 0)
-                return r;
+        tfd = timerfd_create(CLOCK_BOOTTIME_ALARM, TFD_NONBLOCK|TFD_CLOEXEC);
+        if (tfd < 0)
+                return log_error_errno(errno, "Error creating timerfd");
```

Comment:
please always include `%m` to show the error that happened:

```c
if (tfd < 0)
        return log_error_errno(errno, "Failed to create timer fd: %m");
```

or so. (this always uses the error number passed as first arg)